src/App.jsx

```diff
@@ -31,32 +42,61 @@ const App = () => {
 
   const productsList = products.map((product) => {
     return (
-      <div key={product.id}>
-        {product.name} - {`${product.price} kr `}
-        <button
-          data-cy="add-to-order-button"
-          onClick={() => addToOrder(product.id)}
-        >
-          Add to Order
-        </button>
-      </div>
+      <>
+        <ProductsListItem
+          product={product}
+          addToOrder={addToOrder}
+        />
+        <Divider variant="inset" component="li" />
+      </>
     );
   });
 
   return (
     <>
-      <h1>Slowfood</h1>
-      {order && (
-        <button data-cy="view-order" onClick={() => setViewOrder(!viewOrder)}>
-          {viewOrder ? "Hide Order" : "View Order"}
-        </button>
-      )}
-      <h3 data-cy="message-box">{message}</h3>
-      {viewOrder ? (
-        <OrderDetails order={order} />
-      ) : (
-        <div data-cy="product-list">{productsList}</div>
-      )}
+      <Header
+        brand="Material Kit React"
+        fixed
+        color="transparent"
+        changeColorOnScroll={{
+          height: 400,
+          color: "white",
+        }}
+      />
+      <Parallax image={require("./assets/img/bg4.jpg").default}>
+        <div className={classes.container}>
+          <GridContainer>
+            <GridItem>
+              <div className={classes.brand}>
+                <h1 className={classes.title}>Material Kit React.</h1>
+                <h3 className={classes.subtitle}>
+                  A Badass Material-UI Kit based on Material Design.
+                </h3>
+              </div>
+            </GridItem>
+          </GridContainer>
+        </div>
+      </Parallax>
+      <div className={classNames(classes.main, classes.mainRaised)}>
+        <Container maxWidth="sm" className="menuContainer">
+          <h1>Slowfood</h1>
+          {order && (
+            <button data-cy="view-order" onClick={() => setViewOrder(!viewOrder)}>
+              {viewOrder ? "Hide Order" : "View Order"}
+            </button>
+          )}
+          <h3 data-cy="message-box">{message}</h3>
+          {viewOrder ? (
+            <OrderDetails order={order} />
+          ) : (
+            <List
+              data-cy="product-list"
+              sx={{ width: '100%', maxWidth: 360, bgcolor: 'background.paper' }}>
+              {productsList}
+            </List>
+          )}
+        </Container>
+      </div>
```

Comment:
I'm not a MaterialUI expert, but this feels a bit weird. Do you really need to mix `classes.container` on a `div`
 with  `<GridContainer>`?

Reply:
No idea. That's how it has been used in the theme. I did some experiments, but I can try to apply this styling class to the GridContainer and see what happens